streams/src/test/java/org/apache/kafka/streams/integration/KTableKTableForeignKeyJoinIntegrationTest.java

```diff
@@ -140,9 +163,9 @@ public void doJoinFromLeftThenDeleteLeftEntity() {
             final KeyValueStore<String, String> store = driver.getKeyValueStore("store");
 
             // Pre-populate the RHS records. This test is all about what happens when we add/remove LHS records
-            right.pipeInput("rhs1", "rhsValue1");
-            right.pipeInput("rhs2", "rhsValue2");
-            right.pipeInput("rhs3", "rhsValue3"); // this unreferenced FK won't show up in any results
+            right.pipeInput("rhs1", "rhsValue1", baseTimestamp);
```

Comment:
Should we use "auto advance" instead of passing in timestamps expliclity?

Reply:
I'll leave it as is for readability, since I believe auto-advance would be per-topic and the test behavior is clearer this way since it's explicit that records from the two input topics do not overlap/interleave in their timestamps.